app/jobs/itms_job.rb

```diff
@@ -183,4 +183,11 @@ def find_or_create_additional_info_by_json(json, scope)
       scope.find_or_create_by!(nazov: j['nazov'], url: j['url'])
     end
   end
+
+  def recent_data(days: 3)
```

Comment:
upravil by som tuto cast takto:

```ruby
def recent_records
  -> (json) { (json['updatedAt'] || json['createdAt']).to_date > 3.days }
end
```

kludne nech to spadne ak tam oba casy chybaju a potom:

`.select(&recent_records)`

Reply:
ok, to je ovela krajsie